path.py

```diff
@@ -727,6 +727,15 @@ def mkdir(self, mode=0777):
     def makedirs(self, mode=0777):
         os.makedirs(self, mode)
 
+    def makedirs_p(self, mode=0777):
+        try:
+            os.makedirs(self, mode)
+        except OSError, e:
```

Comment:
thanks!!!! fixed. any other feedback more than welcome, or maybe you have another way to fix it.